### PR TITLE
Internal Obligations csv filename

### PR DESCRIPTION
**Pull request by NateHogan**

### CODE REVIEW

File: src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ public async Task<CSVFileData> HandleAsync(GetPcsObligationsCsv message)
 
             var fileContent = csvWriter.Write(authority.Schemes);
             var timestamp = DateTime.Now;
-            var fileName = $"{authority.Abbreviation}_pcsobligationuploadtemplate{timestamp.ToString("dd/MM/yyyy_HHmm")}.csv";
+            var fileName = $"{authority.Abbreviation}_pcsobligationuploadtemplate{timestamp.ToString("_yyyy_dd/MM/yyyy_HHmm")}.csv";
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs:59]: Can we write a test for this....

Comment (NateHogan) [reply src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs:59]: Yes, although the infrastructure for the request handler unit tests is still sitting in my unit tests branch, should I add it there?

Comment (JohnPadgettCivica) [reply src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs:59]: Sounds good

Comment (meglenapnovoselska) [inline src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs:59]: is there a constant class for keeping date and time formats such as this one?
if there is one, would you mind adding it in?

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs (hunk 1 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs:59]: @NateHogan Meg makes a good point if you could do this change in your test branch... i.e. put _yyyy_dd/MM/yyyy_HHmm into a constant somewhere

Comment (NateHogan) [reply src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs]: Good idea, moved the DateTimeConstants class to the core project, and added the format as a constant

**Pull request merged**